kayenta-atlas/src/main/java/com/netflix/kayenta/atlas/model/Backend.java

```diff
@@ -72,12 +78,41 @@
   @Getter
   private String description;
 
+  public List<String> getTargets() {
```

Comment:
Might be worth a sentence or two for future readers to know what is going on in this method.

Reply:
Will add.